frontend2/src/views/QuickStart.tsx

```diff
@@ -1,9 +1,43 @@
 import React from "react";
-import DocumentationPage from "../components/DocumentationPage";
-import { BC23_QUICKSTART } from "../content/bc23";
+import { quickStartText } from "../content/ManageContent";
+import { QuickStartPage } from "../content/ContentStruct";
+import { useEpisodeId } from "../contexts/EpisodeContext";
+import OptionalSectionCardMarkdown from "../components/OptionalSectionCardMarkdown";
 
 const QuickStart: React.FC = () => {
-  return <DocumentationPage text={BC23_QUICKSTART} />;
+  const { episodeId } = useEpisodeId();
+  const currentQuickStartText = quickStartText[episodeId];
+
+  return (
```

Comment:
currently when you go to bc23 -> quickstart theres just an empty screen because obv no content for that year yet. Let's add a `<PageTitle>` as well as some component if we know none of the cards will be rendered? If that ends up being too complicated you could also just do this:

```
episodeId !== DEFAULT_EPISODE && 
  <>
        If you don't see any content here, don't worry! 
        This is a legacy episode, so there may not be any information for this page.
  </>
```